tubemq-client-twins/tubemq-client-go/client/consumer_impl.go

```diff
@@ -120,7 +120,8 @@ func (c *consumer) register2Master(needChange bool) error {
 		}
 		c.master = node
 	}
-	for c.master.HasNext {
+	retryCount := 0
+	for {
 		rsp, err := c.sendRegRequest2Master()
 		if err != nil {
```

Comment:
Why here if get error will return err, but not use next master address to register?

Reply:
If first address net error, will always return error ?